src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -944,6 +945,14 @@ class HttpPutManifestFail : public HttpFake {
   }
 };
 
+int num_events_PutManifestError = 0;
+void process_events_PutManifestError(const std::shared_ptr<event::BaseEvent>& event) {
+  std::cout << event->variant << "\n";
+  if (event->variant == "UpdateCheckComplete") {
+    num_events_PutManifestError++;
```

Comment:
We should check here, that it's really an error.

Reply:
We do that in 947 line

Reply:
Well, there we check a result, not an event. I think it won't hurt to check if event status matches the result.

Reply:
OK, Updated